test/powershell/Language/Operators/ComparisonOperator.Tests.ps1

```diff
@@ -82,6 +82,21 @@ Describe "ComparisonOperator" -Tag "CI" {
         param($lhs, $operator, $rhs)
         Invoke-Expression "$lhs $operator $rhs" | Should -BeFalse
     }
+
+    It "Should be <result> for backtick comparison <lhs> <operator> <rhs>" -TestCases @(
+        @{ lhs = 'abc`def'; operator = '-like'; rhs = 'abc`def'; result = $false }
+        @{ lhs = 'abc`def'; operator = '-like'; rhs = 'abc``def'; result = $true }
+        @{ lhs = 'abc`def'; operator = '-like'; rhs = 'abc````def'; result = $false }
+        @{ lhs = 'abc``def'; operator = '-like'; rhs = 'abc````def'; result = $true }
+        @{ lhs = 'abc`def'; operator = '-like'; rhs = [WildcardPattern]::Escape('abc`def'); result = $true }
+        @{ lhs = 'abc`def'; operator = '-like'; rhs = [WildcardPattern]::Escape('abc``def'); result = $false }
+        @{ lhs = 'abc``def'; operator = '-like'; rhs = [WildcardPattern]::Escape('abc``def'); result = $true }
+        @{ lhs = 'abc``def'; operator = '-like'; rhs = [WildcardPattern]::Escape('abc````def'); result = $false }
```

Comment:
There  is "Win7 backcompatibility requires treating '`' pattern as '' pattern" comment in regex.cs file. Please add a test for the scenario.

Reply:
@iSazonov Do we need to worry about this? I am not sure why we are still trying to support Win 7 compatibility and why such code even exists 😄. Can you elaborate on what scenario this covers?

Reply:
Given also the code changes here are in `Escape()` I am not sure what I am testing. The Win7 compatibility is more with ensuring `WildcardPattern` in `Parse()` method treams lone backtick as empty string.